From c1cfda0a9bddd4a8a1b8daed018399668c0dc480 Mon Sep 17 00:00:00 2001
From: Patrick Barsanti <patrick.barsanti@amarulasolutions.com>
Date: Thu, 23 Jul 2026 18:46:16 +0200
Subject: [PATCH] include: configs: Enlarge SYS_BOOTM_LEN

To avoid the following issue:

Uncompressing LZ4 Kernel Image from 0x05480000 to 0x00400000 ...
Image too large(0x4000000 >= 0x4000000): increase CONFIG_SYS_BOOTM_LEN

Double the SYS_BOOTM_LEN from 64M to 128M

Upstream-Status: Inappropriate [oe-specific]
Signed-off-by: Patrick Barsanti <patrick.barsanti@amarulasolutions.com>
---
 include/configs/rk3588_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/rk3588_common.h b/include/configs/rk3588_common.h
index 67a816a33f..17e8da4321 100644
--- a/include/configs/rk3588_common.h
+++ b/include/configs/rk3588_common.h
@@ -32,7 +32,7 @@
 
 #define CONFIG_SYS_INIT_SP_ADDR		0x00600000
 #define CONFIG_SYS_LOAD_ADDR		0x00600800
-#define CONFIG_SYS_BOOTM_LEN		(64 << 20)	/* 64M */
+#define CONFIG_SYS_BOOTM_LEN		(128 << 20)	/* 128M */
 #define COUNTER_FREQUENCY		24000000
 
 #define GICD_BASE			0xfe600000
